Toolset/libraries/revideextensionlibrary.livecodescript

```diff
@@ -668,6 +668,68 @@ private function __fetchModuleData pExtensionFile
    return tDataA
 end __fetchModuleData
 
+private function __extensionIsBuiltin pID
+   set the itemdelimiter to "."
+   return the number of items in pID is 3
+end __extensionIsBuiltin
+
```

Comment:
Sounds sensible. Is this something we'll need to add to the IDE when it loads widgets so it tracks where the widget was loaded from? I'm happy to review it for now but we should add this to the todo so we don't forget. Thoughts?